frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -169,12 +169,17 @@ private boolean isCodeBlockedForSession(Session session) {
                 session.getEmailAddress(), CODE_BLOCKED_KEY_PREFIX);
     }
 
-    private void blockCodeForSessionAndResetCount(Session session) {
+    private void blockCodeForSession(Session session) {
         codeStorageService.saveBlockedForEmail(
                 session.getEmailAddress(),
                 CODE_BLOCKED_KEY_PREFIX,
                 configurationService.getBlockedEmailDuration());
+        LOG.info("Email is blocked");
```

Comment:
Why do we need this as is? :)

Reply:
just additional logging to help understand what's going on